# Mission A -- pillar pos/vel/acc
start_pos_A : [0.0, 0.0, 0.1]
start_vel_A : [0.0, 0.0, 0.0]
start_acc_A : [0.0, 0.0, 0.0]

mid_pos1_A : [0.0, 0.0, 1.0]
mid_vel1_A : [0.0, 0.0, 0.0]
mid_acc1_A : [0.0, 0.0, 0.0]

mid_pos2_A : [6.5, 2.5, 1.0]
mid_vel2_A : [1.0, 0.0, 0.0]
mid_acc2_A : [0.0, 0.0, 0.0]

mid_pos3_A : [8.0, 2.5, 1.0]
mid_vel3_A : [1.0, 0.0, 0.0]
mid_acc3_A : [0.0, 0.0, 0.0]

mid_pos4_A : [10.0, 2.5, 1.0]
mid_vel4_A : [1.0, 0.0, 0.0]
mid_acc4_A : [0.0, 0.0, 0.0]

end_pos_A : [12.5, 2.5, 1.0]
end_vel_A : [1.0,  0.0, 0.0]
end_acc_A : [0.0, 0.0, 0.0]

# Mission B -- tunnel pos/vel/acc 

start_pos_B : [18.3, -1.5, 1.5]
start_vel_B : [0.5, -0.5, 0.0]
start_acc_B : [0.0, 0.0, 0.0]

mid_pos1_B : [20.1, -2.5, 1.5]
mid_vel1_B : [0.5, -0.5, 0.0]
mid_acc1_B : [0.0, 0.0, 0.0]

end_pos_B : [22.0, -2.0, 1.0]
end_vel_B : [0.0, -1.0, 0.0]
end_acc_B : [0.0, 0.0, 0.0]

# Mission C -- maze pos/vel/acc

start_pos_C : [24.0, -10.0, 1.0]
start_vel_C : [0.0, -1.0, 0.0]
start_acc_C : [0.0, 0.0, 0.0]

mid_pos1_C : [20.0, -15.0, 1.0]
mid_vel1_C : [0.0, -1.0, 0.0]
mid_acc1_C : [0.0, 0.0, 0.0]

mid_pos2_C : [20.0, -20.0, 1.0]
mid_vel2_C : [0.0, -1.0, 0.0]
mid_acc2_C : [0.0, 0.0, 0.0]

end_pos_C : [22.0, -18.0, 1.0]
end_vel_C : [0.0, -1.0, 0.0]
end_acc_C : [0.0, 0.0, 0.0]

# Mission D -- loop pos/vel/acc

start_pos_D : [17.0, -20.0, 1.0]
start_vel_D : [-1.0, 0.0, 0.0]
start_acc_D : [0.0, 0.0, 0.0]

end_pos_D : [13.5, -20.0, 1.5]
end_vel_D : [-1.0, 0.0, 0.0]
end_acc_D : [0.0, 0.0, 0.0]

# Mission E -- pillar pos/vel/acc

start_pos_E : [10.0, -20.0, 1.0]
start_vel_E : [-1.0, 0.0, 0.0]
start_acc_E : [0.0, 0.0, 0.0]

end_pos_E : [2.0, -20.0, 1.0]
end_vel_E : [0.0, 0.0, 0.0]
end_acc_E : [0.0, 0.0, 0.0]

 

